views/storage_detail_widget.php

```diff
@@ -1,25 +1,62 @@
 <div class="col-lg-4">
-    <h4><i class="fa fa-hdd-o"></i> <span data-i18n="disk_report.storage"></span></h4>
-    <table>
-        <tr>
-            <th data-i18n="disk_report.size"></th><td class="mr-TotalSize"></td>
-        </tr>
-        <tr>
-            <th data-i18n="disk_report.used"></th><td class="mr-UsedSize"></td>
-        </tr>
-        <tr>
-            <th data-i18n="disk_report.free"></th><td class="mr-freespace"></td>
-        </tr>
-        <tr>
-            <th data-i18n="disk_report.smartstatus"></th><td class="mr-smartstatus"></td>
-        </tr>
-    </table>
+    <h4 data-i18n="disk_report.storage"></h4>
+    <table id="disk_report_detail" class="table"></table>
 </div>
 
 <script>
+$(document).on('appReady', function(){
+        // Get mdm_status data
```

Comment:
Comment refers to mdm

Reply:
Busted.  

I also fixed the Encryption status output to be localized.